SU-SOE/stanford_soe_helper
# SOE-1970 Postcard linked

#READY FOR REVIEW

# Summary
- Postcard Linked Bean type front-end

# Needed By (Date)
- 8/3 (end of sprint)


# Urgency
- N/A


# Steps to Test

- Pull this branch
- Add a Postcard Linked Bean (ensure that an icon is manually added, as this is a manual step; I can provide the exact arrow icon if needed)
- The display of the blocks will dynamically change based on the span class (or lack thereof) added to the Bean type. Spans 8-12 and no span will result in a horizontal block, and spans 1-7 will result in a vertical block (three examples on the above page)


# Affected Projects or Products
- SoE

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-1970

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
